apps/remix-ide/src/app.js

```diff
@@ -131,7 +131,10 @@ class AppComponent {
       '6fd22d6fe5549ad4c4d8fd3ca0b7816b.mod': 35 // remix desktop
     }
     this.showMatamo = matomoDomains[window.location.hostname] && !Registry.getInstance().get('config').api.exists('settings/matomo-analytics')
-    this.walkthroughService = new WalkthroughService(appManager, this.showMatamo)
+    this.showEnter = !localStorage.getItem('hadUsageTypeAsked')
```

Comment:
For later, you might want to store the type instead of a boolean. So other components are also aware about the current mode/type.